src/actions/Account.actions.js

```diff
@@ -1,17 +1,17 @@
 import { ethers } from "ethers";
-import { addresses, Actions } from "../constants";
+import { addresses, Actions, THEGRAPH_ID } from "../constants";
 import { abi as ierc20Abi } from "../abi/IERC20.json";
-import { abi as OHMPreSale } from "../abi/OHMPreSale.json";
-import { abi as OlympusStaking } from "../abi/OlympusStaking.json";
-import { abi as MigrateToOHM } from "../abi/MigrateToOHM.json";
 import { abi as sOHM } from "../abi/sOHM.json";
 import { abi as sOHMv2 } from "../abi/sOhmv2.json";
-import { abi as LPStaking } from "../abi/LPStaking.json";
-import { abi as DistributorContract } from "../abi/DistributorContract.json";
-import { abi as BondContract } from "../abi/BondContract.json";
-import { abi as DaiBondContract } from "../abi/DaiBondContract.json";
+import { ApolloClient, InMemoryCache, gql } from "@apollo/client";
 
-const parseEther = ethers.utils.parseEther;
+const APIRUL = "https://api.thegraph.com/subgraphs/id/"+THEGRAPH_ID;
+
+const client = new ApolloClient({
```

Comment:
It doesn't make sense to instantiate the ApolloClient in each view's redux actions. We should instantiate it once on startup and have it accesible by a shared lib.

Reply:
you right. this was for testing purposes. Now broken out into src/lib